### PR TITLE
Sharp

### PR DESCRIPTION
**Pull request by Dref360**

### CODE REVIEW

File: deep-learning/_posts/2017-04-11-sharpmask.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+tags:   deep-learning CNN localization bounding-boxes
+author: Frédéric Branchaud-Charron
+pdf:   "https://arxiv.org/pdf/1612.03144.pdf"
+cite:
+  authors: "T. Lin, P. Dollár, R. Girshick, K. He, B. Hariharan, S. Belongie"
+  title:   "Feature Pyramid Networks for Object Detection"
+  venue:   "Computer Vision and Pattern Recognition, 2017. CVPR 2017"
+---
+
+SharpMask was designed as a successor to DeepMask. The main drawbacks from DeepMask were speed and performances.
```
Comment (jodoin) [inline deep-learning/_posts/2017-04-11-sharpmask.md]: add a link to https://vitalab.github.io/deep-learning/2017/04/10/deepmask.html

Comment (jodoin) [inline deep-learning/_posts/2017-04-11-sharpmask.md]: are speed and ...

File: deep-learning/_posts/2017-04-11-sharpmask.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+SharpMask, using a double pyramid layout, is faster and more accurate than DeepMask. Indeed, SharpMask has an inference time of 0.8s where DeepMask needed more than 1.3 seconds to perform its task. The authors used DeepMask for the bottom-up pyramid, but they switched the finals layers, called "head" to be more efficient. They then add a top-down pyramid and laterals connections to make a prediction in the same way as DeepMask.
```
Comment (jodoin) [inline deep-learning/_posts/2017-04-11-sharpmask.md]: needs more.. 
to preform the same task.

File: deep-learning/_posts/2017-04-11-sharpmask.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+---
+layout: review
+title:  "Feature Pyramid Networks for Object Detection"
```
Comment (jodoin) [inline deep-learning/_posts/2017-04-11-sharpmask.md]: SharpMask : Feature...

File: deep-learning/_posts/2017-04-11-sharpmask.md (hunk 1 above)
Comment (jodoin) [inline deep-learning/_posts/2017-04-11-sharpmask.md]: maybe mention that as for DeepMask, SharpMask can be used as an object detection method when combined with R-CNN

Comment (jodoin) [inline deep-learning/_posts/2017-04-11-sharpmask.md]: Also has some sort of UNet shape.  No?

File: deep-learning/_posts/2017-04-11-sharpmask.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+tags:   deep-learning CNN localization segmentation
+author: Frédéric Branchaud-Charron
+pdf:   "https://arxiv.org/pdf/1603.08695.pdf"
+cite:
+  authors: "Pinheiro, P. O., Lin, T. Y., Collobert, R., & Dollár, P."
+  title:   "SharpMask : Learning to refine object segments"
+  venue:   "European Conference on Computer Vision (pp. 75-91). Springer International Publishing."
+---
+
+SharpMask was designed as a successor to [DeepMask]({{ site.baseurl }}{% link deep-learning/_posts/2017-04-10-deepmask.md %}). The main drawbacks from DeepMask are speed and performances. Both can be used as an object detection method when combined with R-CNN. This approach is using the popular [U-Net]({{ site.baseurl }}{% link deep-learning/_posts/2017-02-27-unet.md %}) architecture to combine low and high-level features.
```
Comment (jodoin) [inline deep-learning/_posts/2017-04-11-sharpmask.md]: link to **https://vitalab.github.io/deep-learning/2017/04/18/rcnn.html**

File: deep-learning/_posts/2017-04-11-sharpmask.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+SharpMask, using a double pyramid layout, is faster and more accurate than DeepMask. Indeed, SharpMask has an inference time of 0.8s where DeepMask needs more than 1.3 seconds to perform the same task. The authors used DeepMask for the bottom-up pyramid, but they switched the finals layers, called "head" to be more efficient. They then add a top-down pyramid and laterals connections to make a prediction in the same way as DeepMask.
```
Comment (jodoin) [inline deep-learning/_posts/2017-04-11-sharpmask.md]: final layers

Comment (jodoin) [inline deep-learning/_posts/2017-04-11-sharpmask.md]: lateral connections

File: deep-learning/_posts/2017-04-11-sharpmask.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
+---
+layout: review
+title:  "SharpMask : Learning to Refine Object Segments"
+tags:   deep-learning CNN localization segmentation
+author: Frédéric Branchaud-Charron
+pdf:   "https://arxiv.org/pdf/1603.08695.pdf"
+cite:
+  authors: "Pinheiro, P. O., Lin, T. Y., Collobert, R., & Dollár, P."
```
Comment (lemairecarl) [inline deep-learning/_posts/2017-04-11-sharpmask.md]: The dot at the end is superfluous (there is already one in the template)

File: deep-learning/_posts/2017-04-11-sharpmask.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+  title:   "SharpMask : Learning to refine object segments"
+  venue:   "European Conference on Computer Vision (pp. 75-91). Springer International Publishing."
```
Comment (lemairecarl) [inline deep-learning/_posts/2017-04-11-sharpmask.md]: The dot at the end is superfluous (there is already one in the template)

**Pull request merged**